docs/ops/sequence/AUGRUCell_dev.md

```diff
@@ -0,0 +1,144 @@
+ z - update gate, r - reset gate, h - hidden gate
+ a - attention score
+
+  rt = f(Xt*(Wr^T) + Ht-1*(Rr^T) + Wbr + Rbr)
+  zt = f(Xt*(Wz^T) + Ht-1*(Rz^T) + Wbz + Rbz)
+  ht = g(Xt*(Wh^T) + (rt (.) Ht-1)*(Rh^T) + Rbh + Wbh)  # 'linear_before_reset' is False
+
+  zt' = (1 - at) (.) zt  # multiplication by attention score
+
+  Ht = (1 - zt') (.) ht + zt' (.) Ht-1
+```
+
+Activation functions for gates: *sigmoid* for f, *tanh* for g.
+
+**Attributes**
+
+* *hidden_size*
+
+  * **Description**: *hidden_size* specifies hidden state size.
+  * **Range of values**: a positive integer
+  * **Type**: `int`
+  * **Required**: *yes*
+
+
+The rest of the attributes are supported only with default values:
+
+* *activations*
+
+  * **Description**: activation functions for gates
+  * **Range of values**: any combination of *relu*, *sigmoid*, *tanh*
+  * **Type**: a list of strings
+  * **Default value**: *sigmoid* for f, *tanh* for g
+  * **Required**: *no*
+  * **Supported value**: *sigmoid* for f, *tanh* for g
+
+* *activations_alpha, activations_beta*
+
+  * **Description**: *activations_alpha, activations_beta* attributes of functions; applicability and meaning of these attributes depends on chosen activation functions
+  * **Range of values**: a list of floating-point numbers
+  * **Type**: `float[]`
+  * **Default value**: []
+  * **Required**: *no*
+  * **Supported value**: []
+
+* *clip*
+
+  * **Description**: *clip* specifies bound values *[-C, C]* for tensor clipping. Clipping is performed before activations.
+  * **Range of values**: a positive floating-point number
+  * **Type**: `float`
+  * **Default value**: `0.` that means the clipping is not applied
+  * **Required**: *no*
+  * **Supported value**: `0.`
+
+* *linear_before_reset*
+
+  * **Description**: *linear_before_reset* flag denotes, if the output of hidden gate is multiplied by the reset gate before or after linear transformation.
+  * **Range of values**: True or False
+  * **Type**: `boolean`
+  * **Default value**: False
+  * **Required**: *no*.
+  * **Supported value**: False
+
+**Inputs**
+
+* **1**: `X` - 2D tensor of type *T* and shape `[batch_size, input_size]`, input data. **Required.**
+
+* **2**: `H_t` - 2D tensor of type *T* and shape `[batch_size, hidden_size]`. Input with initial hidden state data. **Required.**
+
+* **3**: `W` - 2D tensor of type *T* and shape `[3 * hidden_size, input_size]`. The weights for matrix multiplication, gate order: zrh. **Required.**
+
+* **4**: `R` - 2D tensor of type *T* and shape `[3 * hidden_size, hidden_size]`. The recurrence weights for matrix multiplication, gate order: zrh. **Required.**
+
+* **6**: `B` - 2D tensor of type *T*. The biases. If *linear_before_reset* is set to  `False`, then the shape is `[3 * hidden_size]`, gate order: zrh. Otherwise the shape is `[4 * hidden_size]` - the sum of biases for z and r gates (weights and recurrence weights), the biases for h gate are placed separately. **Required.**
```

Comment:
can we have two inputs for biases for `before` and `after` separately? We will just have zero biases among them for each case. Now it is not good to have different shapes for each case.

Reply:
I think augrucell is just following grucell here. I prefer to align with ops. :)

Reply:
Agree, let align with `GRUCell` then:)